### ISSUE TITLE
Clay dropdown issue with focus order

### ISSUE THREAD

**Issue by balcevia** [liferay/clay#3914]
## What are the steps to reproduce?
Example https://clayui.com/docs/components/drop-down.html - the problem also occurs

The problem occurs when we use dropdown only with the keyboard. When you enter dropdown with the TAB button, and select a value with Enter, the dropdown closes and the focus jumps to the beginning of the webpage.

## What is the expected result?
Traditional focus behavior should be implemented, after selecting the dropdown value it should collapse and the focus should be transferred to the element that triggers the dropdown

## Environment

| Tech  | Version |
| ----- | ------- |
| Clay  | v2.18.1  |
| Liferay |   7.2.1 |

**Comment by mardrof** [comment]
Hi, I also have the same problem. This issue is critical when I want to pass WCAG's requirements related to keyboard navigation on the page. Does any work to fix this issue is already in progress?

**Comment by matuzalemsteles** [comment]
Hey @balcevia and @mardrof, revisiting this to try to validate the prioritization of this, Well from what I understand this also happens in 3.x right? I'm not sure which behavior we should follow if the focus should go back to the button I have the feeling that this is expected behavior because the same happens when selecting the element using the mouse or when we remove an element from the DOM but I'm not sure, if following the behavior of the Select it would go back to the trigger. @marcoscv-work or @drakonux could you help us here?

**Comment by matuzalemsteles** [comment]
I'm closing this issue, this has been fixed now!
